assets/celo/asset.json

```diff
@@ -0,0 +1,5 @@
+{
+  "caip-19": "???:1/erc20:0x765DE816845861e75A25fCA122bb6898B8B1282a",
+  "symbol": "CELO",
```

Comment:
@antoineherzog , CELO is an ERC20 token but not on Ethereum.  What's the value for the chain-id?